packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/AppConfiguration.kt

```diff
@@ -55,6 +56,22 @@ public interface AppConfiguration {
     public val metadataMode: MetadataMode
     public val syncRootDirectory: String
 
+    /**
+     * The name of this app. This is only used as part of debug headers sent when making
+     * network requests to App Services application.
+     *
+     * If no name was defined, `null` will be returned.
+     */
+    public val appName: String?
+
+    /**
+     * Version of this app. This is only used as part of debug headers sent when making
+     * network requests to the App Services application.
+     *
+     * If no version was defined, `null` will be returned.
```

Comment:
I would just reference `Builder.addVersion` instead of describing the value in more detail.